spec/on_page_spec.rb

```diff
@@ -0,0 +1,73 @@
+require 'spec_helper'
```

Comment:
the last spec will not pass because on_page? will simply return true if user doesn't ask any level of verification.